content/en/docs/setup/_index.md

```diff
@@ -20,83 +20,3 @@ Istio {{< istio_version >}} has been tested with these Kubernetes releases: {{<
 
 Visit our [getting started guide](/docs/setup/getting-started/) to
```

Comment:
How about removing the tip above. We already mention in the getting started page which k8s versions are supported.

Reply:
Instead of having this "Visit" line here, how about just changing the description of the Getting Started page to have the same description? That is:

Download, install, and learn how to evaluate and try Istio’s basic features quickly.